components/sections/decasteljau/content.zh-CN.md

```diff
@@ -1,28 +1,28 @@
 # de Casteljau 算法
 
-要计算贝塞尔曲线，我们必须遍历 `t` 的所有值，从 `0` 到 `1`，计算权重函数，得到 `x/y` 的值。当曲线很复杂时，计算量变得非常巨大。利用 `de Casteljau` 算法（一种几何画法），你可以轻易地用笔和尺画出曲线。
+要计算贝塞尔曲线，我们必须遍历 `t` 的所有值，从 0 到 1，计算权重函数，得到 `x/y` 的值。当曲线很复杂时，计算量变得非常巨大。利用 “de Casteljau” 算法（一种几何画法），你可以轻易地用笔和尺画出曲线。
```

Comment:
Are there commonly spaces in texts like these? I know that in Japanese, there aren't, so if you're adding here "for clarity" then please take them back out and we should file an issue to make sure that CSS takes care of this for us.